src/components/Toaster/Toast/Toast.scss

```diff
@@ -153,6 +140,14 @@ $block: '.#{variables.$ns}toast';
         }
     }
 
+    &__icon-container {
+        flex: 0 0 auto;
+
+        &_not-empty {
```

Comment:
Can be implemented without mod: `&:not(:empty) {padding-right: 8px}`

Reply:
Yes I can.
But complex selector especially with pseudo classes like 'not' are a little bit more expensive.

However, if you want, I will rewrite this place.